x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_stream.tsx

```diff
@@ -193,161 +195,174 @@ export const PackagePolicyInputStreamConfig: React.FunctionComponent<{
                   </EuiFlexItem>
                 );
               })}
-              {/* Advanced section */}
-              {(isPackagePolicyEdit || !!advancedVars.length) && (
-                <Fragment>
-                  <EuiFlexItem>
-                    <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+              {/* Advanced section - always shown since we display experimental indexing settings here */}
```

Comment:
Most of the diff here is whitespace only, so recommend viewing with GitHub's "ignore whitespace" option